### DISCUSSION TITLE
ModuleNotFoundError: No module named 'numba'

### DISCUSSION THREAD

**Discussion by kanishknishar** [openai/whisper#1069] [Q&A]
I updated to Whisper v20230308 and when I tried transcribing a file I got the error ``ModuleNotFoundError: No module named 'numba'``.

I tried two commands and got the same issue each time:
``get-ChildItem | ForEach-Object {Write-Output "Now processing: $($_.Name)"; whisper --fp16 False --output_format "srt" --model base.en $_.FullName; Write-Output "---------------"}``

```whisper --fp16 False --output_format "srt" --model base.en '.\01 - Greece and the Western World.mp4'```

**Comment by jongwook** [comment]
Have you tried

```
pip install numba
```

?

**Comment by kanishknishar** [comment]
Error:



ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.

    numpy<1.24,>=1.18 from https://files.pythonhosted.org/packages/6a/03/ae6c3c307f9c5c7516de3df3e764ebb1de33e54e197f0370992138433ef4/numpy-1.23.5-cp310-cp310-win_amd64.whl (from numba):

        Expected sha256 dbee87b469018961d1ad79b1a5d50c0ae850000b639bcb1b694e9981083243b6

             Got        62997b25ab12616ff5371b997210c53be92d9c4f95228f59549cee6982218b57
